### DISCUSSION TITLE
Arm-based Linux distro in the radar?

### DISCUSSION THREAD

**Discussion by spcaipers-arm** [actions/virtual-environments#5259] [General questions] [answered]
I would like to know if there is any plans to provide ARM-based GH environment. Anything?

**Comment by miketimofeev** [comment, marked as answer]
@spcaipers-arm I'm afraid no plans at the moment. There will be an announcement if this happens

**Reply by jgresham** [reply]
Would running a job on an arm docker container be the best way to achieve this now? Any other suggestions? @miketimofeev

**Reply by panekj** [reply]
> Would running a job on a arm docker container be the best way to achieve this now?

Yes, or running self-hosted runner on arm machine. You can use [Oracle Cloud free tier](https://www.oracle.com/cloud/free/) for that.
